scene/main/node.cpp

```diff
@@ -1364,6 +1364,30 @@ bool Node::has_node(const NodePath &p_path) const {
 	return get_node_or_null(p_path) != nullptr;
 }
 
+Node *Node::get_unique_node(const StringName &p_name) const {
+	if (!p_name) {
+		return nullptr;
+	}
+
+	if (data.owned_unique_nodes.size()) {
+		// Has unique nodes in ownership
+		Node **unique = data.owned_unique_nodes.getptr(p_name);
+		if (!unique) {
+			return nullptr;
+		}
+		return unique
+	} else if (data.owner) {
+		Node **unique = data.owner->data.owned_unique_nodes.getptr(p_name);
+		if (!unique) {
+			return nullptr;
+		}
+		return unique
```

Comment:
This is lacking a semicolon.

Reply:
I apologise for my ignorance, I'll update this PR soon, but what is a dereference in this case?

Reply:
`return *unique;` You need to dereference unique because its currently becoming a pointer to a pointer of a Node and is not a pointer to a Node itself, you need to get the pointer to the Node.

Reply:
Thank you, very gentle and insightful!